rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -2145,6 +2146,55 @@ void rskTxWaitingForSignature_uses_pegoutCreation_rskTxHash_after_rskip_375_acti
         assertEquals(0, provider.getReleaseTransactionSet().getEntries().size());
     }
 
+    @Test()
+    void rskTxWaitingForSignature_override_entry_attempt_after_rskip_375_activation() throws IOException {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        when(activations.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP375)).thenReturn(true);
+
+        BridgeConstants spiedBridgeConstants = spy(BridgeRegTestConstants.getInstance());
+        doReturn(1).when(spiedBridgeConstants).getRsk2BtcMinimumAcceptableConfirmations();
+
+        BridgeStorageProvider provider = mock(BridgeStorageProvider.class);
+        BtcTransaction btcTx = mock(BtcTransaction.class);
+        Set<ReleaseTransactionSet.Entry> releaseTransactionSetEntries = new HashSet<>();
+        Keccak256 pegoutCreationRskTxHash = Keccak256.ZERO_HASH;
+        releaseTransactionSetEntries.add(new ReleaseTransactionSet.Entry(btcTx, 1L, pegoutCreationRskTxHash));
+        when(provider.getReleaseTransactionSet()).thenReturn(new ReleaseTransactionSet(releaseTransactionSetEntries));
+        when(provider.getReleaseRequestQueue()).thenReturn(new ReleaseRequestQueue(Collections.emptyList()));
+
+        TreeMap<Keccak256, BtcTransaction> txsWaitingForSignatures = new TreeMap<>();
+
+        txsWaitingForSignatures.put(pegoutCreationRskTxHash, btcTx);
+        when(provider.getRskTxsWaitingForSignatures()).thenReturn(txsWaitingForSignatures);
+
+        Block executionBlock = mock(Block.class);
+        when(executionBlock.getNumber()).thenReturn(2L);
+
+        BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withBridgeConstants(spiedBridgeConstants)
+            .withProvider(provider)
+            .withExecutionBlock(executionBlock)
+            .withActivations(activations)
+            .build();
+
+        Transaction tx = Transaction
+            .builder()
+            .nonce(NONCE)
+            .gasPrice(GAS_PRICE)
+            .gasLimit(GAS_LIMIT)
+            .destination(Hex.decode(TO_ADDRESS))
+            .data(Hex.decode(DATA))
+            .chainId(Constants.REGTEST_CHAIN_ID)
+            .value(DUST_AMOUNT)
+            .build();
+
+        IllegalStateException exception = assertThrows(IllegalStateException.class, () -> {
```

Comment:
Shouldn't this go above? Like `@Test(expected = IllegalStateException.class)`.

We usually do it that way, not sure if anything changed in JUnit 5

Reply:
Exactly, that way works for JUnit 5 but not JUnit 5